addons/actions.md

```diff
@@ -95,6 +95,26 @@ The Timer object supports the following methods:
 * `hasTerminated`: returns true if the code has run and completed
 * `reschedule(AbstractInstant instant)`: reschedules the timer to execute at the new time. If the Timer has terminated this method does nothing.
 
+### Thing Status Action
+
+`getThingStatusInfo(String thingUid)`: Gets ThingStatusInfo of the given thing identified by thingUid and returns the result as ThingStatusInfo
+
+The result is of type ThingStatusInfo. So in theory you can use its functions and fields.
+If you just want to know the status, you can use `thingStatusInfo.getStatus().toString()` and the result will be one of the values in [Thing Status]({{base}}/concepts/things.html).
+
+> If the thing is removed or it's not added yet, it'll return null.
+
+For example:
+```java
+var statusInfo = getThingStatusInfo("zwave:device:c5155aa4:node2")
+
+if ((statusInfo != null) && (statusInfo.getStatus().toString() == "ONLINE")) {
+    logInfo("ThingStatus", "The thing is online.")
+} else {
+    logError("ThingStatus", "The thing isn't online.")
+}
+```
```

Comment:
Yes. I intend to say "isn't online". It can be "offline" or "not in the system".